spec/controllers/works_controller_spec.rb

```diff
@@ -303,6 +303,68 @@
       end
     end
 
+    context "when only some file uploads are deleted for an existing Work with uploads" do
+      let(:uploaded_file1) do
+        fixture_file_upload("us_covid_2019.csv", "text/csv")
+      end
+
+      let(:uploaded_file2) do
+        fixture_file_upload("us_covid_2020.csv", "text/csv")
+      end
+
+      let(:bucket_url) do
+        "https://example-bucket.s3.amazonaws.com/"
+      end
+
+      let(:deleted_uploads) do
+        {
+          work.pre_curation_uploads.first.key => "1",
+          work.pre_curation_uploads[1].key => "0",
```

Comment:
Are there three uploads?  I am seeing upload_file2 and upload_file1.  And a zero means delete, or does a 1 mean delete.